torch/serialization.py

```diff
@@ -338,7 +338,13 @@ def save(obj, f, pickle_module=pickle, pickle_protocol=DEFAULT_PROTOCOL, _use_ne
            containing a file name
         pickle_module: module used for pickling metadata and objects
         pickle_protocol: can be specified to override the default protocol
-
+        _use_new_zipfile_serialization: saves the object as a zip file rather than a binary file
+    
+    Note:
+        The file extension does not matter while saving the object. The object can be saved using
+        .pt, .pth, .zip or any other extension. You can literally save the object as .py file and 
```

Comment:
If possible, I propose to avoid recommending/suggesting `*.pth` as reasonable extension for serialization, because "pth" is a special file extension for Python and can lead no nasty package importing errors: https://github.com/pytorch/pytorch/issues/14864